src/dotnet/ToolPackage/ToolPackageInstaller.cs

```diff
@@ -133,7 +134,11 @@ internal class ToolPackageInstaller : IToolPackageInstaller
 
             var tempProjectContent = new XDocument(
                 new XElement("Project",
-                    new XAttribute("Sdk", "Microsoft.NET.Sdk"),
+                    new XElement("PropertyGroup",
```

Comment:
Nit: maybe a comment explaining why this needs to come before the `Import` and not with the `PropertyGroup` below?

Reply:
Fixed https://github.com/dotnet/cli/pull/9509/files#diff-514fdb6bf1ab21b1a21adfd2ccf82145R139

Reply:
👍

Reply:
fixed https://github.com/dotnet/cli/pull/9509/files#diff-514fdb6bf1ab21b1a21adfd2ccf82145R139